include/seqan3/argument_parser/detail/version_check.hpp

```diff
@@ -429,15 +430,18 @@ class version_checker
         return "powershell.exe -NoLogo -NonInteractive -Command \"& {Invoke-WebRequest -erroraction 'silentlycontinue' -OutFile";
 #else  // Unix based platforms.
         if (!system("wget --version > /dev/null 2>&1"))
-            return "wget -q -O";
+            return "wget --timeout=10 --tries=1 -q -O";
         else if (!system("curl --version > /dev/null 2>&1"))
-            return "curl -o";
-    #ifndef __linux  // ftp call does not work on linux
-        else if (!system("which ftp > /dev/null 2>&1"))
-            return "ftp -Vo";
-    #endif // __linux
-        else
+            return "curl --connect-timeout 10 -o";
+    // In case neither wget nor curl is available try ftp if system is OpenBSD or FreeBSD.
```

Comment:
```suggestion
    // In case neither wget nor curl is available try ftp/fetch on OpenBSD/FreeBSD.
    // which are always included with the respective base system.
```